src/modules/networkcfg/main.py

```diff
@@ -53,4 +53,13 @@ def run():
             except FileExistsError:
                 pass
 
+    # We need to overwrite the default resolv.conf in the chroot.
+    source_resolv = "/etc/resolv.conf"
+    target_resolv = os.path.join(root_mount_point, "etc/resolv.conf")
+    if os.path.exists(source_resolv):
```

Comment:
Fair enough. I think it doesn't matter much either way unless one does static configuration, since NetworkManager happily messes around with `resolv.conf` anyway.